### PR TITLE
Standardize all strings to use double quotes for consistency

### PR DESCRIPTION
**Pull request by HrushiYadav**
Updated all string literals to use double quotes as per the new style guide. Exceptions for strings within strings using single quotes have been handled accordingly. This change improves code readability and ensures consistency across the project.

Fix: #299

### CODE REVIEW

File: scripts/run.py [hunk 1]
```diff
@@ -97,9 +97,9 @@
 for neuron in neurons:
     # Construct the PM2 start command
     command = (
-        f"pm2 start {neuron['file']} --interpreter python3 --name {neuron['hotkey']}:{neuron['type']} --"
-        + f" --wallet.name {coldkey} --wallet.hotkey {neuron['hotkey']} --subtensor.network {network} --netuid {netuid}"
-        + f" --axon.port {neuron['port']} --logging.debug {neuron.get('config')}"
+        f"pm2 start {neuron["file"]} --interpreter python3 --name {neuron["hotkey"]}:{neuron["type"]} --"
+        + f" --wallet.name {coldkey} --wallet.hotkey {neuron["hotkey"]} --subtensor.network {network} --netuid {netuid}"
+        + f" --axon.port {neuron["port"]} --logging.debug {neuron.get("config")}"
```
Comment (dbobrenko) [inline scripts/run.py:102]: This code will result in error with our target python versions 3.9, 3.10

**Review by dbobrenko — CHANGES_REQUESTED** [review]
Thanks for your contribution! Left some comments to make sure the code is not broken

File: scripts/run.py (hunk 1 above)
Comment (HrushiYadav) [reply scripts/run.py:102]: I'll Revert this part

Comment (dbobrenko) [reply scripts/run.py:102]: @HrushiYadav thanks! Also make sure to make the change against pre-staging branch, or even: https://github.com/macrocosm-os/prompting/tree/SN1-150-remove-low-value-subcompetitions

It's a new upcoming change with lots of changes and refactor done

**Comment by cassova** [comment]
I think this PR may no longer be necessary with the introduction of black.

File: scripts/run.py (hunk 1 above)
Comment (HrushiYadav) [reply scripts/run.py:102]: I'll commit to same branch

**Comment by Hollyqui** [comment]
Yup, unfortunately made redundant by SN1-155 and SN1-150

**Pull request closed without merging**